### PR TITLE
React fontawesome #bb372

### PR DESCRIPTION
**Pull request by prabalsingh24**
### Problem
https://tickets.metabrainz.org/browse/BB-372
Migrating React-FontAwesome : entities/footer.js

### Areas of Impact
src/client/components/pages/entities/footer.js

### CODE REVIEW

File: src/client/components/pages/entities/footer.js [hunk 1]
```diff
@@ -41,22 +41,22 @@ function EntityFooter({deleted, entityUrl, lastModified}) {
 							href={`${entityUrl}/edit`}
 							title="Edit Entity"
 						>
-							<Icon name="pencil"/>&nbsp;Edit
+							<FontAwesomeIcon icon="pencil"/>&nbsp;Edit
```
Comment (MonkeyDo) [inline src/client/components/pages/entities/footer.js]: pencil -> pencil-alt

File: src/client/components/pages/entities/footer.js [hunk 2]
```diff
@@ -41,22 +41,22 @@ function EntityFooter({deleted, entityUrl, lastModified}) {
...
+							<FontAwesomeIcon icon="history"/>&nbsp;History
 						</Button>
 						<Button
 							bsStyle="danger"
 							disabled={deleted}
 							href={`${entityUrl}/delete`}
 							title="Delete Entity"
 						>
-							<Icon name="remove"/>&nbsp;Delete
+							<FontAwesomeIcon icon="remove"/>&nbsp;Delete
```
Comment (MonkeyDo) [inline src/client/components/pages/entities/footer.js]: remove -> times

**Comment by prabalsingh24** [comment]
I think I screwed up the rebase thing :(

**Comment by MonkeyDo** [comment]
> I think I screwed up the rebase thing :(

Not screwed up, no. It's a bit harder to review as is, but not the end of the world, especially for these small localized changes.
There are two possible workflows to update a PR with changes on master: merge commit and rebase. You did a merge commit here, which works just fine.

The rebase requires a force-push, which can make working in teams more difficult. Hence why some people prefer the merge commit workflow, which makes sure not to rewrite the history.

However, if you don't have a clear need to update the PR with the changes on master (for example, PR 2 requires changes from PR 1, and needs updating once PR 1 is merged), it's always better to abstain :)

In any case, the history will be clean when I merge this, which is ultimately the goal.

**Pull request merged**